includes/class-wc-payments-order-service.php

```diff
@@ -883,6 +890,28 @@ private function mark_order_held_for_review_for_fraud( $order, $intent_data ) {
 		$this->set_intention_status_for_order( $order, $intent_data['intent_status'] );
 	}
 
+	/**
+	 * Given the charge, adds the application_fee_amount from the charge to the given order as metadata.
+	 *
+	 * @param WC_Order               $order The order to update.
+	 * @param WC_Payments_API_Charge $charge The charge to get the application_fee_amount from.
```

Comment:
Nitpick: Moving from https://github.com/Automattic/woocommerce-payments/pull/5852/files#r1154097958, at least, we might update this doc block. 

```suggestion
	 * @param WC_Order                    $order The order to update.
	 * @param null|WC_Payments_API_Charge $charge The charge to get the application_fee_amount from.
```